### PR TITLE
Convert Google Doc Meeting Minutes to Markdown

### PR DESCRIPTION
**Pull request by jsoref**
Requested https://openssf.slack.com/archives/C01AAV2TTEG/p1703799986181569

Meeting minutes sourced from:
https://docs.google.com/document/d/1706vJpuyq4NpHpVYsOTeU90j5RpoJREX7MRlhAo-CW4/edit

Contents were as of "December 29, 11:57 AM"

Markdown generated using:
https://workspace.google.com/marketplace/app/docs_to_markdown/700168918607

Markdown split using:
```perl
perl -ne '
if (/^# \*\*(\d{4}-\d\d-\d\d)/) {
    if ($file) {
        close $fh;
    }
    $file="$1.md";
    open $fh, ">", $file;
    s/\s*\\$/**/;
    print $fh $_;
} elsif ($file) {
    s/^# /## /;
    s/^Agenda$/## Agenda/;
    print $fh $_;
}
' 2022.md
```

### CODE REVIEW

File: minutes/2022-11-29.md [hunk 1]
```diff
@@ -0,0 +1,145 @@
+# **2022-11-29 Meeting - CANCELLED**
+2022-11-15 Meeting \
+**Attendance (please add yourself):
```
Comment (jsoref) [inline minutes/2022-11-29.md:3]: This wasn't handled correctly, something about the export confused things...

**Review by hythloda — APPROVED** [review]
Thanks a ton!

File: minutes/2023-10-17.md [hunk 2]
```diff
@@ -0,0 +1,246 @@
...
+    * **INFORM **- With the upcoming Operational Model discussions with the GB, the TAC will be updating our TI definitions, lifecycle stages, and associated documentation within [https://github.com/ossf/tac/tree/main/process](https://github.com/ossf/tac/tree/main/process) 
+    * 
+        * TIs will be _Working Groups, Projects_ (expanded now to cover specs), and _SIGs_
+        * All TIs will share a common lifecycle (that will be tailored for each specific category as needed) - _Sandbox, incubating, Graduated, Archived_
+        * 
+
+<p id="gdcalert7" ><span style="color: red; font-weight: bold">>>>>>  gd2md-html alert: inline image link here (to images/image7.png). Store image on your image server and adjust path/filename/extension if necessary. </span><br>(<a href="#">Back to top</a>)(<a href="#gdcalert8">Next alert</a>)<br><span style="color: red; font-weight: bold">>>>>> </span></p>
+
+
+![alt_text](images/image7.png "image_tooltip")
```
Comment (jsoref) [inline minutes/2023-10-17.md:212]: The image will need to be added later (@hythloda said that wouldn't be a problem)

**Review by hythloda — APPROVED** [review]
LGTM

File: minutes/2023-11-14.md [hunk 3]
```diff
@@ -0,0 +1,396 @@
...
+<p id="gdcalert5" ><span style="color: red; font-weight: bold">>>>>>  gd2md-html alert: inline image link here (to images/image5.png). Store image on your image server and adjust path/filename/extension if necessary. </span><br>(<a href="#">Back to top</a>)(<a href="#gdcalert6">Next alert</a>)<br><span style="color: red; font-weight: bold">>>>>> </span></p>
+
+
+![alt_text](images/image5.png "image_tooltip")
+
+
+<p id="gdcalert6" ><span style="color: red; font-weight: bold">>>>>>  gd2md-html alert: inline image link here (to images/image6.png). Store image on your image server and adjust path/filename/extension if necessary. </span><br>(<a href="#">Back to top</a>)(<a href="#gdcalert7">Next alert</a>)<br><span style="color: red; font-weight: bold">>>>>> </span></p>
+
+
+![alt_text](images/image6.png "image_tooltip")
```
Comment (jsoref) [inline minutes/2023-11-14.md:386]: These images will need to be manually added later.

**Review by jsoref — COMMENTED** [review]
These are the only problems we know about, and my understanding is that they aren't showstoppers.

**Review by hythloda — APPROVED** [review]

**Pull request merged**